### ISSUE TITLE
App doesn't load - wmic command not found - Windows10

### ISSUE THREAD

**Issue by snowcoiner** [DeFiCh/app#364]
### Preflight Checklist

- [x] I have searched [DeFiChain’s Github Issue Page](https://github.com/DeFiCh/app/issues) and none matches my issue
- [x] I have read through [How To Submit Issues for DeFi App](https://github.com/DeFiCh/app/wiki/How-to-submit-issues-for-DeFi-app”)

### Issue Details

- **Current DeFi App Version:**
  - 2.1.12
- **Previous DeFi App Version:**
  -  Last working Version: 2.1.9
- **Operating System (OS):**
  - Windows 10 (Version 10.0.19042.746)

### Screenshot(s) / screen recording(s) of issue

![image](https://user-images.githubusercontent.com/77891624/105607140-7c348f00-5d9d-11eb-9883-dfc224673fc1.png)

### Actual Behavior

1. Open the app
2. The loading icon/circle does not appear

### App log / binary log / app config
Main Log from "C:\Users\XXX\AppData\Roaming\defi-app\logs"
`[2021-01-23 16:50:44.496] [IPC-RENDERER-info] [Starting App] App is running with version 2.1.12

[2021-01-23 16:50:50.898] [IPC-RENDERER-error] [getConfig] TypeError: Cannot read property 'success' of undefined
[2021-01-23 16:51:12.686] [IPC-RENDERER-info] Stopping DeFi Process Manager
[2021-01-23 16:51:12.826] [IPC-RENDERER-error] Der Befehl "wmic" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.`

**Comment by juldivers** [comment]
I have exactly the same problem

**Comment by izzycsy** [comment]
Hi @snowcoiner , update [v2.1.15](https://defichain.com/downloads/) is out.

Do give a thumbs up to this comment if your issue has been resolved, otherwise, provide new information (include screenshots) and app logs. Thank you.

**Comment by snowcoiner** [comment]
I've had the same issue with v2.1.15.
`[2021-01-31 14:38:24.734] [IPC-RENDERER-info]  "[Starting App] App is running with version 2.1.15"
[2021-01-31 14:38:32.125] [IPC-RENDERER-error] [getConfig] "Cannot read property 'message' of undefined"
`

After reading the above mentioned solution I was able to get it to run.
I've moved the entry "%SystemRoot%\System32\Wbem" to the top of the system variable "Path" and the defid process was able to start.
